e2e/features/layers/recent-layers-mobile-test.js

```diff
@@ -0,0 +1,53 @@
+const { assertCategories } = require('../../reuseables/layer-picker.js');
+const skipTour = require('../../reuseables/skip-tour.js');
+const {
+  addLayers,
+  collapsedLayerButton,
+  layersSearchField,
+  layerSearchList,
+  layerPickerBackButton,
+} = require('../../reuseables/selectors.js');
+
+const TIME_LIMIT = 10000;
+
+module.exports = {
+  before: (c) => {
+    skipTour.loadAndSkipTour(c, TIME_LIMIT);
+    c.url(`${c.globals.url}?t=2020-07-04`);
+    c.resizeWindow(375, 667); // iPhone 6/7/8 dimensions
```

Comment:
```suggestion
    c.resizeWindow(375, 667); // iPhone 6/7/8 dimensions
    c.waitForElementVisible(collapsedLayerButton, TIME_LIMIT);
```
I was getting some failed local tests at this stage of the resize. May need to be added elsewhere for mobile changes.

Reply:
Thanks, I'll take a look!  Tests seem to be passing locally for me but looks like Travis is failing as of the last commit on this branch. 🤔

Reply:
Ah I see, it's Firefox specific.  Nice find!